archive/c/c/capitalize.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+
+char *captialize(char str[]) {
+    for(int i = 0; i < strlen(str); i++) {
+        if(i == 0) {
+            str[i] = toupper(str[i]);
+        } else {
+            continue;
+        }
+    }
+    return str;
+}
+
+int main(int argc, char *argv[]) {
+    if(argc == 2) {
+        printf("Result: %s\n", captialize(argv[1]));
+    } else if(argc > 2) {
+        printf("Use quotes around multiple strings.\n");
+    } else {
+        printf("Expected an argument.\n");
+    }
+}
```

Comment:
The requirements call out that there should be a check to verify the the input is a string and not a number. Can that check be added here?

Reply:
Disregard.

Reply:
This is probably a good opportunity to put together some formal tests in the documentation. I've put it off because @auroq usually helps us write those, but we can do that!